### DISCUSSION TITLE
Golden Soroban Stellar Quest NFT Submission

### DISCUSSION THREAD

**Discussion by artgypsy** [stellar/sorobanathon#128] [General]
The Journey 
https://litemint.com/items/GBFJ43V7ICYGKWJPFIFTXTHK3NN34KSRVB6XO37ZXOQVR4CQ4FODQQIK/JNEY

The Temple
https://litemint.com/items/GCRNHHQQHQNB3PPPUSRXKCUCY7YQUQORUXAKABKH3QI7QF4JEWTYAJ5F/TEMP

Lava Snakes
https://litemint.com/items/GCEDEWKMHEGLYMYM77NOPIQYK7T2YHGLPACZFN5MQD6WRTP36RVR5JLM/LAVA

The Potion Room

https://litemint.com/items/GCPDIBRZIXVZMDCAU7LI7ZUYD5JUBCFPEQBRJZOWILQLQ6JE7N62KUUA/TION

The Gatekeeper
https://litemint.com/items/GBDNRV6RVHJWLQMMLW7D4ZYOM23D35R22L75O3IW62KGF4CXLETXS4XL/GATE

The Golden Soroban
https://litemint.com/items/GAVVZLBLPTLQ7WH3IVLGMB6BD56DJ3C4VR5E2F662MOVH53K56WQ7F2A/SORO

The Disappearance 
https://litemint.com/items/GBASC4XGHZLJTEEZZXWMNFPB7WAR2ZV4MLR2IC655F2UKFBRXEXHVY7X/DISA

**Comment by stellarjohn** [comment]
Hi [artgypsy](https://github.com/artgypsy)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by artgypsy** [reply]
Thank you! That’s amazing 🤩

**Comment by artgypsy** [comment]
John, did you receive my Kyc form? Just making sure. Merry Christmas 🎁🎄

**Comment by artgypsy** [comment]
John, did you receive my form? I haven’T received anything yet and most
everyone I know has. Just want to verify you received my Kyc? Thank you 🙏

**Comment by stellarjohn** [comment]
Hey, we have not received a KYC from from you. Please try to resubmit.

**Reply by stellarjohn** [reply]
Also be sure to fully submit the typeform. You should end on a screen like this. 
![image](https://user-images.githubusercontent.com/98538609/210403244-9b349c3d-2159-4f8a-8f8e-fe98b1420518.png)

**Comment by artgypsy** [comment]
Ok ✅ complete. Please let me know if you did not receive it.

**Reply by stellarjohn** [reply]
Received!

**Comment by artgypsy** [comment]
Thank you!

**Comment by artgypsy** [comment]
Hey there, just checking when those of us who have not received our XLM can
expect the dispersement?

**Reply by stellarjohn** [reply]
Everyone should be receiving it today. Thank you for your patience!

**Comment by artgypsy** [comment]
Great, thanks for the update!

**Comment by artgypsy** [comment]
Hi there just checking in to see if there were any updates?

**Comment by artgypsy** [comment]
I have not received anything yet. Can you please let me know?

**Comment by artgypsy** [comment]
Is there a problem with my submission? Please let me know as you said we
would be paid on Friday.

**Reply by stellarjohn** [reply]
There's no problem with your submission. Sorry for the delay. Our account department is getting the payments out asap.

**Comment by artgypsy** [comment]
It is concerning that it’s taking this long. Thank you

**Comment by artgypsy** [comment]
Is stellar going to pay us before the next is out?

**Comment by artgypsy** [comment]
Hello,
Please give us all an update that are waiting to be paid for our nft work
on Soroban. This is not a good look for Stellar and does not instill trust
with artists that are wiling to participate in these quests. If artists do
not get paid they will lose trust in the blockchain and go elsewhere. It’s
vitally important that we support Stellar artists and keep them in the
ecosystem. They deserve honest communication and to be paid within a
reasonable amount of time. Please respond to everyone inquiring of the
status of their Soroban XLM payment.

**Reply by stellarjohn** [reply]
I'm very sorry about the delay. The accounting department is still working on getting this round of payments out. I will get you information about when the disbursement will go out as soon as I have it.
